app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,80 @@
+<div id="header">
```

Comment:
I actually deleted this file, you'll want to put the link to the Admin Dashboard in `/app/views/shared/_dashboard_nav_links.html.erb`